lib/pointing_party_web/controllers/session_controller.ex

```diff
@@ -0,0 +1,26 @@
+defmodule PointingPartyWeb.SessionController do
+  use PointingPartyWeb, :controller
+  alias PointingParty.User
+
+  def new(conn, _params) do
+    changeset = User.changeset(%User{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, params) do
+    changeset = User.changeset(%User{}, params["user"])
```

Comment:
Might be nice to refactor this a bit to hide the calls to `User.changeset`, `changeset.valid?` and `Ecto.Changeset.apply_changes`, maybe wrap them up in some helper functions in the User module?